nav2_mppi_controller/include/nav2_mppi_controller/tools/utils.hpp

```diff
@@ -276,8 +265,12 @@ inline bool withinPositionGoalTolerance(
 template<typename T>
 auto normalize_angles(const T & angles)
 {
-  auto theta = xt::eval(xt::fmod(angles + M_PIF, 2.0f * M_PIF));
-  return xt::eval(xt::where(theta < 0.0f, theta + M_PIF, theta - M_PIF));
+  return (angles + M_PIF).unaryExpr([&](const float x){
+             float remainder = std::fmod(x, 2.0f * M_PIF);
+             return remainder < 0.0f ? remainder + M_PIF : remainder - M_PIF;
+             });
+  // auto theta = angles - (M_PIF * ((angles + M_PIF_2) * (1.0f / M_PIF)).floor());
```

Comment:
Is this comment still worthwhile / tested for speed?

Reply:
yes, I had tested it for the speed, was not better so removed it now.